source/isaaclab/isaaclab/managers/event_manager.py

```diff
@@ -384,7 +384,7 @@ def _prepare_terms(self):
             self._mode_term_cfgs[term_cfg.mode].append(term_cfg)
 
             # check if the term is a class
-            if isinstance(term_cfg.func, ManagerTermBase):
+            if isinstance(term_cfg.func, type(ManagerTermBase)):
```

Comment:
```suggestion
            if inspect.isclass(term_cfg.func):
```

Reply:
This makes sense. I'll make the change to use the inspect module. Since the code base is quite complex, I did not want to make an assumption about the check and  chose to stick with using `isinstance`. If the intent of the check is to simply check if it's a class vs a function, this suggestion is better.